apps/ui/components/application/ApplicantInfoPreview.tsx

```diff
@@ -1,86 +1,88 @@
 import React from "react";
 import { useTranslation } from "next-i18next";
 import { ApplicantTypeChoice, type ApplicationQuery } from "@gql/gql-types";
-import { getTranslation } from "common/src/common/util";
-import { SpanTwoColumns, TwoColumnContainer } from "../common/common";
-import Address from "./AddressPreview";
-import { StyledLabelValue } from "./styled";
+import {
+  ApplicationInfoContainer,
+  InfoItemContainer,
+  InfoItem,
+} from "./styled";
 
+const LabelValue = ({
+  label,
+  value,
+}: {
+  label: string;
+  value?: string | number | null;
+}) => {
+  return (
+    <InfoItemContainer
+      className={
+        label === "preview.organisation.billingAddress" ? "fullWidth" : ""
+      }
+    >
+      <InfoItem>
+        <h4 className="info-label">{label}</h4>
+        <p>{value}</p>
+      </InfoItem>
+    </InfoItemContainer>
+  );
+};
 type Node = NonNullable<ApplicationQuery["application"]>;
 export function ApplicantInfoPreview({
   application,
 }: {
   application: Node;
 }): JSX.Element {
   const { t } = useTranslation();
-
+  const applicant = {
+    firstName: application.contactPerson?.firstName,
+    lastName: application.contactPerson?.lastName,
+    email: application.contactPerson?.email,
+    phoneNumber: application.contactPerson?.phoneNumber,
+  };
+  const contactPersonName = `${applicant.firstName} ${applicant.lastName}`;
+  const contactInfo = `${applicant.phoneNumber} / ${applicant.email}`;
+  const addressString = `${application.organisation?.address?.streetAddressFi}, ${application.organisation?.address?.postCode} ${application.organisation?.address?.cityFi}`;
+  const billingAddressString = `${application.billingAddress?.streetAddressFi}, ${application.billingAddress?.postCode} ${application.billingAddress?.cityFi}`;
```

Comment:
this doesn't look correct. how does it make sure these are not `undefined`?
so we don't end up with 
> "undefined, undefined undefined"

also how is the billing address checked if it exists? turning it into string with no null checks makes it impossible to check later